vsphere/tests/test_api.py

```diff
@@ -1,13 +1,57 @@
 # (C) Datadog, Inc. 2019-present
 # All rights reserved
 # Licensed under Simplified BSD License (see LICENSE)
+import ssl
+
 import pytest
+import six
 from mock import ANY, MagicMock, patch
 from pyVmomi import vim, vmodl
 
 from datadog_checks.vsphere.api import APIConnectionError, VSphereAPI
 from datadog_checks.vsphere.config import VSphereConfig
 
+if six.PY3:
+    EXPECTED_PROTOCOL = ssl.PROTOCOL_TLS_CLIENT
+else:
+    EXPECTED_PROTOCOL = ssl.PROTOCOL_TLS
```

Comment:
why not always select this one?

ssl.PROTOCOL_TLS_CLIENT
> Auto-negotiate the highest protocol version like PROTOCOL_TLS, but only support client-side SSLSocket connections. 

ssl.PROTOCOL_TLS
> Selects the highest protocol version that both the client and server support.

Reply:
Updated in favour of `ssl.PROTOCOL_TLS`.